### DISCUSSION TITLE
BlogCategory Widget

### DISCUSSION THREAD

**Discussion by houseofsails** [arthelokyo/astrowind#390] [Ideas]
Does anyone know how easy it would be to create a blog category widget, so I could display blog posts in one category on the homepage?

I have tried and failed to do this. 

It seems it should be relatively straightforward but I can't for the life of me figure out how to get the widget to display posts by category on the homepage in the same manner as the Latest Posts widget.

I just thought I'd ask if anyone has already tackled this, as I really love the theme but I need multiple categories to consider using it on my blog.

**Comment by masheeeen** [comment]
Hey @houseofsails,

I am also searching for a solution to this. Have you found any answers yet?
